src/index.ts

```diff
@@ -1,6 +1,10 @@
 import { BackendModule, ReadCallback, Services } from 'i18next'
 
-type ResourceFetcher = () => Promise<Record<string, string>>
+type ResourceFetcher = () => Promise<{
+  __esModule?: true
```

Comment:
Wondering on the double underscore?

Reply:
That it's global vs just private?

Reply:
Is our usage consistent?

Reply:
This just represents how Babel/TypeScript transpile code:

- https://www.typescriptlang.org/tsconfig#esModuleInterop
- https://babeljs.io/docs/en/babel-plugin-transform-modules-commonjs#example